javascript/react/src/hooks/downloads.ts

```diff
@@ -56,7 +57,12 @@ export function useDownloadZip(
 		}
 
 		if (tables.length) {
-			const files = tables
+			const map = new Map<string, number>()
+			const renamedTables = tables.map(t => ({
+				...t,
+				name: renameDuplicatedFileName(map, t.name || t.id || 'table.csv'),
+			}))
+			const files = renamedTables
 				.map(table => tableFile(table))
 				.filter(f => f !== null)
```

Comment:
`!== null` is better expressed as `!= null` 

The reasoning here is that !== is explicitly checking for the null value, while != null will capture null and undefined.